Project_Starting_Point/Server/Server.cpp

```diff
@@ -17,8 +20,69 @@ StorageTypes RxData[7];
 void UpdateData(unsigned int, float);
 float CalcAvg(unsigned int);
 
+class LatencyTimer {
```

Comment:
Looks good. In our chat, I mentioned that I was considering having 2 classes for this implementation. One class should record the latency for a **single** event Maybe called `SingleLatencyEvent`. Another class will be its container (so it'll have the `vector` of the single latency event class) (`LatencyRecorder`). This pattern will help with multithreading later.